src/screens/Shared/ViewBranchMachine/components/TabPaymentsReceived/index.tsx

```diff
@@ -0,0 +1,246 @@
+import { Button, Col, Row, Table } from 'antd';
+import { ColumnsType } from 'antd/lib/table';
+import { RequestErrors, TableHeader, TimeRangeFilter } from 'components';
+import {
+	ViewTransactionModal,
+	formatDateTime,
+	getFullName,
+	useCollectionReceipts,
+	CollectionReceipt,
+	ViewCollectionReceiptModal,
+} from 'ejjy-global';
+import {
+	DEFAULT_PAGE,
+	DEFAULT_PAGE_SIZE,
+	pageSizeOptions,
+	refetchOptions,
+	timeRangeTypes,
+	transactionStatuses,
+} from 'global';
```

Comment:
Some of the variables in here already exists sa ejjy-global. Please use those instead.